apps/wallet/src/ui/app/hooks/useTransactionDryRunWithoutSigner.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
for new files we dont need the Mysten license

Reply:
this file is not used anywhere and will be removed  🙈